lib/index.ts

```diff
@@ -174,6 +266,14 @@ export class Front {
 
 		return newPath;
 	}
+
+	private validateEventSignature(data: any, signature: string): boolean {
+		const hash = crypto.createHmac('sha1', this.apiSecret)
+			.update(JSON.stringify(data))
```

Comment:
Well, what do you know, it can on a TypeError (which makes sense, circular dependency). `try/catch` added.